src/etl/ReportingETL.h

```diff
@@ -81,6 +81,12 @@ class ReportingETL
     {
         std::string ip;
         int port;
+
+        // TODO: remove when clang catches up to gcc/msvc in regards to
+        // emplace_back with brace initialization (c++20)
+        ClioPeer(std::string ip, int port) : ip{ip}, port{port}
+        {
+        }
```

Comment:
note: this can be avoided by using `push_back` at the callsite bu that would be 1 move vs. inplace creation - not very important difference considering this is parsed/created once. happy to leave this or do `push_back` instead - just shout.